### ISSUE TITLE
Update TestRail with a11y case on Rx renewal success

### ISSUE THREAD

**Issue by heredia-jeffrey** [department-of-veterans-affairs/va.gov-team#129530]
### Summary
Update TestRail to contain a11y test case for successful Rx renewal. The success modal after a Rx renewal wasn't properly in focus and should be when completing a request. The details of this change can be found in [this ticket](https://github.com/department-of-veterans-affairs/vets-website/issues/40331)
